pkg/runtime/runtime_manifest_test.go

```diff
@@ -406,6 +406,95 @@ install:
 	assert.Equal(t, []string{"mysql-password", "password"}, gotSensitiveValues, "Incorrect values were marked as sensitive")
 }
 
+func TestResolveStep_DependencyMappedOutput(t *testing.T) {
```

Comment:
I think having a test leveraging golden files like [we did here](https://github.com/getporter/porter/blob/main/pkg/cnab/testdata/bundle-depsv2.json) would be helpful :)

Reply:
@schristoff Are you thinking about changing this test, or adding another?
I'm asking because it feels a little wrong to me to change this one, because it is about generating a single part of the runtime manifest, and the other tests in this file follow the same pattern as used here.

Reply:
Adding another is fine (always needs more tests)